metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "netmanagers-dnsmasq",
```

Comment:
Oh, that's a good point. It's probably worth bumping a version and releasing after we get all these changes completed.